src/script/components/ParticipantItemContent/ParticipantItemContent.tsx

```diff
@@ -80,7 +82,7 @@ export const ParticipantItemContent = ({
               selfString={selfString}
               showAvailability={showAvailabilityState && selfInTeam}
             >
-              {!isSelectable && renderParticipantBadges?.(participant)}
+              {!isSelectable && <UserVerificationBadges user={participant} groupId={groupId} />}
```

Comment:
Instead of a `renderProps` we now inject the `groupId` that can be used to compute the verification badges for e2ei